py_configuration_files/credentials.py

```diff
@@ -42,10 +42,18 @@
 EMAIL_BACKEND = "django.core.mail.backends.filebased.EmailBackend"
 EMAIL_FILE_PATH = "/tmp/credentials-emails"
 
-DEFAULT_FILE_STORAGE = os.environ.get("DEFAULT_FILE_STORAGE", "django.core.files.storage.FileSystemStorage")
+defaultfile_storage = os.environ.get("DEFAULT_FILE_STORAGE")
```

Comment:
I'm honestly kind of shocked that we ever had this set up to be something that you can get from environment variable, but sure, why not. That being said, be consistent on variable name, 

```suggestion
default_file_storage = os.environ.get("DEFAULT_FILE_STORAGE")
```

Reply:
@deborahgu  I get it, and I understand your point about consistency as well.

My change came from what is already there in [credentials/devstack.py](https://github.com/openedx/credentials/blob/master/credentials/settings/devstack.py#L46) file, as this file is almost a copy of it.

Keeping things same so it's easier to compare when things change.